tests/pecan64.ed.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <pecan>
-  <outdir>pecan</outdir>
+  <outdir>/home/carya/pecan_tests/pecan64.ed.xml</outdir>
```

Comment:
can you change this back to pecan (or pecan.ed) so it will work on any machine and not on just those that have a pecan_test folder as user carya.